pwndbg/gdblib/nearpc.py

```diff
@@ -54,7 +54,10 @@ def ljust_padding(lst):
     "nearpc-lines", 10, "number of additional lines to print for the nearpc command"
 )
 show_args = pwndbg.gdblib.config.add_param(
-    "nearpc-show-args", True, "show call arguments below instruction"
+    "nearpc-show-args", True, "whether to show call arguments below instruction"
+)
+show_opcodes_size = pwndbg.gdblib.config.add_param(
+    "nearpc-opcodes-size", 0, "number of bytes of opcodes to print for each instruction"
```

Comment:
nit: "number of opcode bytes to print for each instruction"

Reply:
And maybe the option name should be `nearpc-opcode-bytes`? Not sure.

Reply:
or: `nearpc-show-opcode-bytes`?

Reply:
Ok, I'll use `nearpc-show-opcode-bytes`

Reply:
`nearpc-show-opcode-bytes` makes it sound like a boolean. `nearpc-num-opcode-bytes` is another alternative.

Reply:
> `nearpc-show-opcode-bytes` makes it sound like a boolean. `nearpc-num-opcode-bytes` is another alternative.

Ok, I used `nearpc-num-opcode-bytes` now.